stylesheets/components/toast.css

```diff
@@ -0,0 +1,30 @@
+@import url(/stylesheets/main.css);
+
+.toast-msg {
+     padding: 0.5rem 1rem;
+     background-color: var(--white);
+     border: 0.1rem solid var(--black);
+     font-size: larger;
+   }
+
+.box {
+     display: flex;
+     flex-direction: row;
+     justify-content: center;
+     align-items: center;
+     text-align: center;
+    justify-content: center;
```

Comment:
you are using flex and text-align: center and by default flex-direction is row any specific reason here for using it.

Reply:
oh ya ty. i ll delete it.